definitions/npm/next_v9.x.x/flow_v0.104.x-/test_next_v9.x.x.js

```diff
@@ -127,12 +127,12 @@ export default class TestDocument extends Document {
 }
 
 export class TestApp extends App {
-  static async getInitialProps({ Component, router, ctx }: AppInitialProps) {
+  static async getInitialProps({ Component, router, ctx }: AppInitialProps): Promise<*> {
```

Comment:
Should we change these all to use `any` instead of `*` since it's deprecated? ref: https://flow.org/en/docs/types/utilities/#toc-existential-type
```suggestion
  static async getInitialProps({ Component, router, ctx }: AppInitialProps): Promise<any> {
```